src/swap/SwapActionScreen.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import { SwapScreenSection } from 'src/swap/SwapScreen'
+
+function SwapActionScreen() {
```

Comment:
I'm not a fan of the `SwapActionScreen` name because it is referring to how a user gets to this screen (through the home actions) which doesn't have anything specifically do do with the component itself. In the future there may be other ways that a user can get to the swap screen and then this name will be confusing. Maybe naming this screen simply `SwapScreen` and naming the existing component `SwapDrawerScreen` to differentiate?

Reply:
Good call, renamed the components

Reply:
Renaming existing screens have implications on analytics right? With this change, we'd start logging "SwapScreen" for the screen reached from the actions whereas previously this would refer to the screen from the drawer

Reply:
Okay, I kept the drawer screen's name as `SwapScreen` and the non-drawer is now called `SwapStartScreen`.